releases.yml

```diff
@@ -17,6 +17,10 @@ releases:
           rpm: 136275
         release_jira: ART-10210
         upgrades: 4.15.18,4.15.19,4.15.20,4.15.21,4.15.22,4.15.23,4.16.0,4.16.1,4.16.2,4.16.3
+      issues:
+        exclude:
+        - id: OCPBUGS-35780  # Shipped in 4.16.2
+        - id: OCPBUGS-35780  # Shipped in 4.16.2
```

Comment:
oh we have a repeat

Reply:
I updated my local but forgot to push, my bad

updated